2020/7xxx/CVE-2020-7383.json

```diff
@@ -1,18 +1,92 @@
 {
-    "data_type": "CVE",
-    "data_format": "MITRE",
-    "data_version": "4.0",
     "CVE_data_meta": {
+        "ASSIGNER": "cve@rapid7.con",
+        "DATE_PUBLIC": "2020-10-15T11:23:00.000Z",
```

Comment:
Is this date (in the future) correct?

Reply:
yeah this wants to be Oct 14, if you please, just going by the release note date.

Reply:
@todb-r7 What was the rule for the date made public? Was it when it was reported to us? I put tomorrow's date as that's when the release notes will be going out including the update on this fix.